test/Header/HeaderWrapTest.php

```diff
@@ -84,16 +84,10 @@ public function testMimeDecoding()
     public function testCanBeEncoded()
     {
         // @codingStandardsIgnoreStart
-        $name    = 'Subject';
         $value   = "[#77675] New Issue:xxxxxxxxx xxxxxxx xxxxxxxx xxxxxxxxxxxxx xxxxxxxxxx xxxxxxxx, tähtaeg xx.xx, xxxx";
-        $encoded = "Subject: =?UTF-8?Q?[#77675]=20New=20Issue:xxxxxxxxx=20xxxxxxx=20xxxxxxxx=20?=\r\n =?UTF-8?Q?xxxxxxxxxxxxx=20xxxxxxxxxx=20xxxxxxxx,=20t=C3=A4htaeg=20xx.xx,=20xxxx?=";
         // @codingStandardsIgnoreEnd
         //
         $res = HeaderWrap::canBeEncoded($value);
         $this->assertTrue($res);
-
-        $header = new GenericHeader($name, $value);
-        $res = $header->toString();
-        $this->assertEquals($encoded, $res);
```

Comment:
This last setup/assertion check is just a proxy for a call to zend-mime, that is affected by https://github.com/zendframework/zend-mail/issues/183#issuecomment-351525410
Is this really necessary?

Reply:
Without this check, build fails only on PHP 7.2 with lowest/locked dependencis because of old zend-validator.
Latest depencencies on PHP 7.2 are fine.